examples/generic_connector_configs/README.md

```diff
@@ -16,6 +20,7 @@ requires to authenticate.
 |---|---|---|
 |[GitHub API](https://developer.github.com/v3/)|[github_secretless.yml](./github_secretless.yml)|<ul><li>Edit the supplied configuration to get your [GitHub OAuth token](https://developer.github.com/v3/#oauth2-token-sent-in-a-header) from the correct provider/path.</li><li>Run Secretless with the supplied configuration</li><li>Query the GitHub API using `http_proxy=localhost:8081 curl api.github.com/{request}`</li></ul>|
 |[Elasticsearch API](https://www.elastic.co/guide/en/elasticsearch/reference/current)|[elasticsearch_secretless.yml](./elasticsearch_secretless.yml)|<ul><li>Edit the supplied configuration to get your Elasticsearch [Api Key](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html) or [OAuth token](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-token.html)</li><li>Run secretless with the supplied configuration(s)</li><li>Query the Elasticsearch API using `http_proxy=localhost:9020 curl <Elasticsearch Endpoint URL>/{Request}`</li></ul>
+|[Splunk API](https://docs.splunk.com/Documentation/Splunk/8.0.2/Security/UseAuthTokens)|[splunk_secretless.yml](./splunk_secretless.yml)|<ul><li>Edit the supplied configuration to get your [Splunk authentication token](https://docs.splunk.com/Documentation/Splunk/8.0.2/Security/EnableTokenAuth) from the correct provider/path</li><li>Add the SplunkDefaultServerCert host name to the IP address you are making the curl request to</li><li>Add the target's CA to [Secretless's trusted certificate pool](https://docs.secretless.io/Latest/en/Content/References/connectors/scl_handlers-https.htm#Manageservercertificates)</li><li>Run Secretless with the supplied configuration</li><li>Query the Splunk API using `http_proxy=localhost:8081 curl {instance host name or IP address}:{management port}/{route}` - note that you do not preface your instance host name with `https://`; Secretless will ensure the final connection to the backend server uses SSL.</li></ul>|
```

Comment:
@izgeri @sgnn7 thoughts?

Reply:
I think I agree, let's simplify these steps as much as possible - maybe add a link to the splunk docs on configuring certificates and just ask the user to set up the certs as documented

It would be nice if this PR had a step-by-step set of instructions for how we got it working locally, though, so that if someone else is trying to do that they can find their way back here and understand how to set it up (including adding the default docker image host to /etc/hosts, etc)